### PR TITLE
{lib}[GCCcore/12.2.0] GEOS v3.11.1, googletest v1.12.1, nlohmann_json v3.11.2, ...

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 6 out of 9 (4 easyconfigs in total)
himem04.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/7eb0317c25f17925d83ccd298ddd36f7 for a full test report.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.11.1-GCC-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
```
Comment (branfosj) [inline easybuild/easyconfigs/g/GEOS/GEOS-3.11.1-GCC-12.2.0.eb:1]: See https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/g/GEOS/GEOS-3.10.3-GCC-11.3.0.eb

Comment (verdurin) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.11.1-GCC-12.2.0.eb:1]: @branfosj would you prefer that @jfgrimm changes this to match the earlier one?

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in total)
himem01.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/59523530344becda89587227a95189c2 for a full test report.

File: easybuild/easyconfigs/g/GEOS/GEOS-3.11.1-GCC-12.2.0.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/g/GEOS/GEOS-3.11.1-GCC-12.2.0.eb]: I've switched it to use cmake, which fixed that build failure

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
himem01.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1a57cfd1cd6cb9b038d488eeb6e3a0c9 for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/515a2a00fbd1aefc72ca37ab0cdb35f8 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**